### ISSUE TITLE
Resurrector ignores failed vm

### ISSUE THREAD

**Issue by akshaymankar** [cloudfoundry/bosh#1725]
We use turbulence to test our 4 node deployment.
Turbulence deleted vm, but BOSH did not restart it.

Here is output of `scan and fix` task. 
```
16:05:43 | Scanning 0 VMs: Checking VM states (00:00:00)
16:05:43 | Scanning 0 VMs: 0 OK, 0 unresponsive, 0 missing, 0 unbound (00:00:00)
```

with debug:
```
UPDATE "tasks" SET "state" = 'done', "timestamp" = '2017-06-28 16:05:43.753424+0000', "description" = 'scan and fix', "result" = 'scan and fix complete', "output" = '/var/vcap/store/director/tasks/216', "checkpoint_time" = '2017-06-28 16:05:43.718630+0000', "type" = 'cck_scan_and_fix', "username" = 'bosh_admin', "deployment_name" = 'etcd-kill-vm-non-tls', "started_at" = '2017-06-28 16:05:43.718569+0000', "event_output" = '{"time":1498665943,"stage":"Scanning 0 VMs","tags":[],"total":2,"task":"Checking VM states","index":1,"state":"started","progress":0}
{"time":1498665943,"stage":"Scanning 0 VMs","tags":[],"total":2,"task":"Checking VM states","index":1,"state":"finished","progress":100}
{"time":1498665943,"stage":"Scanning 0 VMs","tags":[],"total":2,"task":"0 OK, 0 unresponsive, 0 missing, 0 unbound","index":2,"state":"started","progress":0}
{"time":1498665943,"stage":"Scanning 0 VMs","tags":[],"total":2,"task":"0 OK, 0 unresponsive, 0 missing, 0 unbound","index":2,"state":"finished","progress":100}
```

Manual cloud-check did find the error and recreated VMs.

**Comment by cppforlife** [comment]
@akshaymankar what version of director do you have?

**Comment by akshaymankar** [comment]
It is `262.0.0 (00000000)`

**Comment by akshaymankar** [comment]
We deleted the deployment completely, then recreated it. The resurrector had failed to bring up the dead VM.

We deleted the deployment completely, then recreated it with a different name. This time the resurrector was able to bring up the dead VM.

**Comment by dpb587-pivotal** [comment]
Closing - this should be fixed via #1737 and is in [v263+](https://github.com/cloudfoundry/bosh/releases/v263).
